pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -632,10 +632,10 @@ func _ipSecReplacePolicyInFwd(params *IPSecParameters, proxyMark bool, dir netli
 }
 
 func ipSecReplacePolicyIn(params *IPSecParameters) error {
-	if err := _ipSecReplacePolicyInFwd(params, true, netlink.XFRM_DIR_IN); err != nil {
+	if err := _ipSecReplacePolicyIn(params, true, netlink.XFRM_DIR_IN); err != nil {
 		return err
 	}
-	return _ipSecReplacePolicyInFwd(params, false, netlink.XFRM_DIR_IN)
```

Comment:
Commit `ipsec: use appropriate name for policy in creation function`:

I'm all for small commits, but I'm going to have to agree with Gray on this one: this is just bugging reviewers when we review the previous commit and notice the function wasn't renamed then. Plus, extracting those three lines doesn't exactly simplify the previous commit.

Reply:
This one was not done for the same reason as splitting the first two commits. This, i just saw later on after I finished the bulk of this change. I'll collapse this. 👍

Reply:
Fixed with latest push